src/Chapter2.hs

```diff
@@ -740,7 +775,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains n = filter (n `elem`)
```

Comment:
In this particular case, you don't need to apply function in infix notation. You can just partially apply it as it is :slightly_smiling_face: 
```suggestion
contains n = filter (elem n)
```